Editor/AGS.Editor/Panes/ScriptEditor.cs

```diff
@@ -56,6 +56,19 @@ public ScriptEditor(Script scriptToEdit, AGSEditor agsEditor, Action<Script> sho
             this.Resize += new EventHandler(ScriptEditor_Resize);
         }
 
+        public string GetScriptTabName()
+        {
+            if (_roomNumber > 0)
+            {
+                UnloadedRoom room = Factory.AGSEditor.CurrentGame.FindRoomByID(_roomNumber);
+                if(room != null && room.Number > 0 && !string.IsNullOrEmpty(room.Description))

```

Comment:
`room.Number > 0` condition is wrong here, because room number 0 is allowed.

Reply:
Uhm, probably `room.Number == _roomNumber` is the correct condition!

Reply:
fixed it in this push https://github.com/adventuregamestudio/ags/compare/49ca91d1d367bb35d6b0a7db9feff1b948423ebe..56ad1d6b2d3e0e5edc52c4d4f94f36c03e441544